service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C29RecoveryOfAChildDocumentParameterGeneratorTest.java

```diff
@@ -269,52 +252,58 @@ void shouldGenerateDocumentWithAllActionsPermittedOfficerNameEnteredAndIsExParte
+            + "of a Care order made on %s.%s", LA_NAME, FORMAT_LOCAL_DATE_TO_STRING, PARAGRAPH_BREAK);
 
-        return format(
-            "The Court is satisfied that %s has parental responsibility for the %s by virtue of %s made on %s.%s",
-            LA_NAME,
-            childOrChildren,
-            orderGrammar + order.getLabel(),
-            FORMAT_LOCAL_DATE_TO_STRING,
-            paragraphBreak);
     }
 
-    private String getEntryMessage(int numberOfChildren) {
-        String childrenGrammar = numberOfChildren > 1 ? "children" : "child";
+    private String getStandardMessageWithEPO() {
+        return format("The Court is satisfied that %s has parental responsibility for the child by virtue "
+            + "of an Emergency protection order made on %s.%s", LA_NAME, FORMAT_LOCAL_DATE_TO_STRING, PARAGRAPH_BREAK);
+    }
+
+    private String getEntryMessage() {
         return format("The court authorises %s to enter the premises known as %s, "
-                + "and search for the %s, using reasonable force if necessary.%s",
+                + "and search for the child, using reasonable force if necessary.%s",
             OFFICER_NAME,
             REMOVAL_ADDRESS.getAddressAsString(", "),
-            childrenGrammar,
-            paragraphBreak
+            PARAGRAPH_BREAK
+        );
+    }
+
+    private String getInformMessage() {
+        return format(
+            "The court requires any person who has information about where the child is, "
+            + "or may be, to give that information to a police constable or an officer of the court, "
+            + "if asked to do so.%s", PARAGRAPH_BREAK
         );
     }
 
-    private String getInformMessage(ManageOrdersEventData eventData) {
-        String policeAssignee = isNull(eventData.getManageOrdersOfficerName())
-            ? "a police constable"
-            : eventData.getManageOrdersOfficerName();
+    private String getInformMessageWithPoliceOfficer() {
         return format(
             "The court requires any person who has information about where the child is, "
-            + "or may be, to give that information to %s or an officer of the court, "
-            + "if asked to do so.%s", policeAssignee, paragraphBreak
+                + "or may be, to give that information to " + OFFICER_NAME + " or an officer of the court, "
+                + "if asked to do so.%s", PARAGRAPH_BREAK
         );
     }
 
     private String getProduceMessage() {
         return format(
             "The court directs that any person who can produce the child when asked to by %s to do so.%s",
-            OFFICER_NAME, paragraphBreak
+            OFFICER_NAME, PARAGRAPH_BREAK
         );
     }
 
     private String getRemoveMessage() {
-        return format("The Court authorises %s to remove the child.%s", OFFICER_NAME, paragraphBreak);
+        return format("The Court authorises %s to remove the child.%s", OFFICER_NAME, PARAGRAPH_BREAK);
```

Comment:
same for these:
REMOVE_MESSAGE = format("The Court authorises %s to remove the child.%s", OFFICER_NAME, PARAGRAPH_BREAK);